### ISSUE TITLE
New fields are added to the inputed core table and not to a duplicate

### ISSUE THREAD

**Issue by dkav** [linkagescape/linkage-mapper#61]
Reviewing how new core related data are added to the core table, I discovered that the new fields are added to the inputed core table and not to a duplicate. While this seems to be the intent, based on the documentation, I assumed that the fields would have been added to a duplicate.

**Comment by johngallo** [comment]
So the task at some point is to have the model make a copy of the cores file, probably in the outputs folder, maybe a "Copied Inputs" sub-folder or .gdb to be explicit, and add the new fields to that one.   The time consuming part will be finding all the downstream dependencies to this change, I am not sure how well a search and replace will work, but maybe this is easier than I anticipate.  In the meantime, low priority seems like the right priority level.

I recommend that the resistance surface and other inputs are also copied here. 

One of the benefits to this is that to archive and or share results from a model run, it will be nice to just need the outputs folder...Thinking/discussion along these lines should happen before implementing where/how/what features to copy..

**Comment by johngallo** [comment]
I am trying to figure out why the product is not looking quite right, and am setting parameter values. As part of this, I am looking at the relative value of linkages based on individual criteria. I am seeing At another time: Rel_Perm and Rel_Close in the LCPPaths table, but not anything about the average Core Area Value between the two cores getting connected, nor their normalized values (e.g. Avg_CAV and Rel_CAV fields). Are these values being calculated still? if so, great! And would it be time consuming to add these values to the table? 

Maybe this is the field that Darren was referring to, that is added to the input, but not to the copy in the outputs folder?  If not, this comment needs to become its own issue.

**Comment by dkav** [comment]
Closing with the opening of #121.
